docs/resources/equinix_ecx_l2_connection.md

```diff
@@ -184,8 +184,11 @@ for more details.
 docs.
 
 * `additional_info` - (Optional) one or more additional information key-value objects
-  * `name` - (Required) additional information key
+  * `key` - (Required) additional information key
```

Comment:
Does this apply for ecx_l2 though? Looking at the schema in it this won't apply to the Fabric v3 APIs.